src/Storefront/Resources/views/storefront/layout/header/actions/cart-widget.html.twig

```diff
@@ -1,8 +1,6 @@
 {% block layout_header_actions_cart_widget %}
     <span class="header-cart-icon">
-        {% sw_icon 'bag' style {
-            ariaLabel: 'checkout.cartTitle'|trans
-        } %}
+        {% sw_icon 'bag' %}
```

Comment:
Should the label stay here, because it is a icon only button? 🤔

Reply:
No, actually this is redundant because the upper element has the `aria-label`:

![image](https://github.com/user-attachments/assets/063f826b-6a95-4346-bec6-0edca8abce91)

Imo the actual button, anchor or other interactive element should have the label and never the icon, unless the icon is really "standalone" without any parent element represents the "action".